infoschema/slow_log.go

```diff
@@ -125,23 +135,33 @@ func ParseSlowLog(tz *time.Location, scanner *bufio.Scanner) ([][]types.Datum, e
 }
 
 type slowQueryTuple struct {
-	time         time.Time
-	txnStartTs   uint64
-	user         string
-	connID       uint64
-	queryTime    float64
-	processTime  float64
-	waitTime     float64
-	backOffTime  float64
-	requestCount uint64
-	totalKeys    uint64
-	processKeys  uint64
-	db           string
-	indexIDs     string
-	isInternal   bool
-	digest       string
-	statsInfo    string
-	sql          string
+	time              time.Time
+	txnStartTs        uint64
+	user              string
+	connID            uint64
+	queryTime         float64
+	processTime       float64
+	waitTime          float64
+	backOffTime       float64
+	requestCount      uint64
+	totalKeys         uint64
+	processKeys       uint64
+	db                string
+	indexIDs          string
+	isInternal        bool
+	digest            string
+	statsInfo         string
+	numCopTasks       uint64
+	avgProcessTime    float64
+	p90ProcessTime    float64
+	maxProcessTime    float64
+	maxProcessAddress string
+	AvgWaitTime       float64
```

Comment:
```suggestion
	avgWaitTime       float64
```
ditto for variables below

Reply:
done. PTAL